exampleCourse/questions/template/multiple-choice/images/server.py

```diff
@@ -0,0 +1,28 @@
+import random
+
+
+def generate(data):
+    # Create a list with the file name of UIUC figures.
+    uiuc_figures = [
+        "fig1.jpeg",
+        "fig2.jpeg",
+        "fig3.jpeg",
+        "fig4.jpeg",
+        "fig5.jpeg",
+        "fig6.jpeg",
+        "fig7.jpeg",
+    ]
+
+    # Shuffle the order of the UIUC figures.
+    random.shuffle(uiuc_figures)
+
+    # Store the first three shuffled UIUC figures as the wrong answers.
+    data["params"]["wrong_filename0"] = uiuc_figures[0]
```

Comment:
```suggestion
    data["params"]["wrong_filename0"] = uiuc_figures.pop()
```
If you want to do this non-destructively, you can create an iterator and call the `next` function.

Reply:
Yeah, I think that would be the cleanest.

Reply:
I agree it's cleaner but I don't think we should use mustache iteration here. The pattern of just hardcoding options 1 through 4 is much easier to understand, I think, and we should be optimizing for comprehension by non-coders. In general, anything we can do to make the code simpler (e.g., avoiding loops) is a win.

Reply:
This discussion is moot because `server.py` has been removed completely in favor of having `<pl-multiple-choice>` randomize things for us.